Examples/Image/Classification/AlexNet/Read_labelclsloc.py

```diff
@@ -20,4 +20,3 @@ def readlabel(xmlfilename):
     return clsname
 if __name__ == "__main__":
     name = readlabel(sys.argv[1])
```

Comment:
Currently, there's no indication to the user that the script has done the reading. Can we add some thing to address that, e.g. print the labels that were read.